lib/builders/latexmk.js

```diff
@@ -62,10 +62,20 @@ export default class LatexmkBuilder extends Builder {
     const enableShellEscape = atom.config.get('latex.enableShellEscape')
     const enableSynctex = atom.config.get('latex.enableSynctex') !== false
     const engine = this.getLatexEngine(filePath)
+    const cleanExtensions = atom.config.get('latex.cleanExtensions')
 
-    if (shouldRebuild) {
-      args.push('-g')
+    switch (action) {
+      case 'rebuild':
+        args.push('-g')
+        break
+      case 'clean':
+        args.push('-c')
+        if (cleanExtensions.length) {
+          args.push(`-e "\\$clean_ext='${cleanExtensions.join(' ')}'"`)
+        }
+        break
```

Comment:
A deep clean and then a rebuild can be done with the `-gg` option. We could add a command for that.